ocw/lib/gce.py

```diff
@@ -56,6 +56,16 @@ def _delete_resource(self, api_call, resource_name, *_, **kwargs) -> None:
             self.compute_client().routes: "route",
             self.compute_client().subnetworks: "subnetwork",
         }.get(api_call, "resource")
+
+        # Get object details / metadata and check the labels
+        resource_details = api_call().get(**kwargs).execute()
+        labels = resource_details.get('labels', {})
+        if labels:
+            self.log_dbg(f"Resource {resource_type}/{resource_name} has these labels: {labels}")
+            if 'pcw_ignore' in labels and labels["pcw_ignore"] == "1":
```

Comment:
documentation explicitly stating that resources should have tag **with any value**  ( just grep README for "any value" https://github.com/SUSE/pcw/blob/master/README.md ) 
```suggestion
            if 'pcw_ignore' in labels:
```